src/order/conditionally_complete_lattice.lean

```diff
@@ -707,6 +707,9 @@ open_locale classical
 
 variables [conditionally_complete_linear_order_bot α]
 
+theorem cInf_le' {s : set α} {a : α} (h : a ∈ s) : Inf s ≤ a :=
+cInf_le ⟨⊥, λ a _, bot_le⟩ h
```

Comment:
It looks like this theorem is in the wrong namespace - how about putting it in the previous section, about 10 lines up?